addons/web/controllers/main.py

```diff
@@ -367,7 +367,7 @@ def xml2json_from_elementtree(el, preserve_whitespaces=False):
     return res
 
 class HomeStaticTemplateHelpers(object):
-    """
+    r"""
```

Comment:
Here too (and other places). It's a docstring, not a regex? Maybe worth changing `/!\` into `!!!` to indirectly simply bypass it in such a case?